pkg/schema/openapi/v2/schema.go

```diff
@@ -1,7 +1,6 @@
-package openapi
+package v2
 
-func getSchema() []byte {
-	return []byte(`
+var schema = []byte(`
```

Comment:
Can you please rename title (line number 5) to "OpenAPI schema". They're not the same anymore.